api-server/migrations/20210509145547-add-soft-delete.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+      ALTER TABLE venues
+        ADD COLUMN is_soft_deleted BOOLEAN
```

Comment:
Does Sequelize not provide drop-in /opt-in support for soft deletes?

Reply:
nah, I doubt it. It's not that tightly integrated. It would need control of your schema for that, but we control the schema through simple migrations.